app/views/results.nunjucks

```diff
@@ -23,16 +23,25 @@
     {% include "includes/search-form.nunjucks" %}
 
   {% else %}
-
-    <a href="{{ SITE_ROOT }}" class="link-back">Back</a>
-    <h1 class="sr-only" data-results="{{gps.length}}">Search results</h1>
-    <h2 class="results__header">GP {% if gps.length === 1 %}surgery{% else %}surgeries{%endif %} matching '{{ search }}' </h2>
-    <p>If you can't see your surgery, check that you're using the right name and try searching again.</p>
+    <h1 class="sr-only" data-results="{{gps.length}}" data-term="{{ search }}">Search results</h1>
+    {% if gps.length === 1 %}
```

Comment:
Can this logic be pulled back away from the view? Perhaps so there is a new property available, something like `resultsHeader`?